unison-cli/src/Unison/Cli/ProjectUtils.hs

```diff
@@ -273,106 +275,3 @@ expectRemoteProjectBranchByTheseNames = \case
 remoteProjectBranchDoesntExist :: ProjectAndBranch ProjectName ProjectBranchName -> Cli void
 remoteProjectBranchDoesntExist projectAndBranch =
   Cli.returnEarly (Output.RemoteProjectBranchDoesntExist Share.hardCodedUri projectAndBranch)
-
-------------------------------------------------------------------------------------------------------------------------
```

Comment:
These were all just moved to `Unison.Project.Util` in parser-typechecker so they're available in the local ui server too.